spring-kafka-3/src/main/java/com/baeldung/spring/kafka/start/stop/consumer/UserEventListener.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.spring.kafka.start.stop.consumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserEventListener {
+    private static final Logger logger = LoggerFactory.getLogger(UserEventListener.class);
```

Comment:
Can we add spaces after the class declaration in this file and others?

Reply:
Added space after class declarations